cob_base_drive_chain/package.xml

```diff
@@ -13,33 +13,32 @@
 
   <buildtool_depend>catkin</buildtool_depend>
   <build_depend>roscpp</build_depend>
-  <build_depend>sensor_msgs</build_depend>
   <build_depend>std_msgs</build_depend>
   <build_depend>std_srvs</build_depend>
+  <build_depend>sensor_msgs</build_depend>
   <build_depend>diagnostic_msgs</build_depend>
   <build_depend>control_msgs</build_depend>
+  <build_depend>message_generation</build_depend>
+  <build_depend>cob_utilities</build_depend>
   <build_depend>cob_generic_can</build_depend>
   <build_depend>cob_canopen_motor</build_depend>
-  <build_depend>cob_utilities</build_depend>
-  <build_depend>message_generation</build_depend>
   <build_depend>libpcan</build_depend>
   <build_depend>libntcan</build_depend>
-
   
   <run_depend>roscpp</run_depend>
-  <run_depend>sensor_msgs</run_depend>
   <run_depend>std_msgs</run_depend>
   <run_depend>std_srvs</run_depend>
+  <run_depend>sensor_msgs</run_depend>
   <run_depend>diagnostic_msgs</run_depend>
   <run_depend>control_msgs</run_depend>
+  <run_depend>message_runtime</run_depend>  
+  <run_depend>cob_utilities</run_depend>
   <run_depend>cob_generic_can</run_depend>
   <run_depend>cob_canopen_motor</run_depend>
-  <run_depend>cob_utilities</run_depend>
-  <run_depend>message_runtime</run_depend>  
   <run_depend>libpcan</run_depend>
   <run_depend>libntcan</run_depend>
   
   <export>
-    <cpp lflags="-Wl,-rpath,${prefix}/common/lib -L${prefix}/common/lib -lcob_base_drive_chain" cflags="-I${prefix}/common/include"/>
+    <cpp cflags="-I${prefix}/common/include" lflags="-Wl,-rpath,${prefix}/common/lib -L${prefix}/common/lib -lcob_base_drive_chain"/>
```

Comment:
Do we need those flags at anymore? I guess this is a leftover from `rosbuild`...